Documentation~/preferences.md

```diff
@@ -1,279 +1,109 @@
 
 ## Snap Settings
 
-Use these properties to customize how snapping behaves with ProBuilder.
+Customize how snapping behaves with ProBuilder.
 
-![Toolbar section](images/prefs_snapping.png)
-
-<table>
-<thead>
-<tr>
-<th colspan="2"><strong>Property:</strong></th>
-<th><strong>Description:</strong></th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td colspan="2"><strong>Snap As Group</strong></td>
-<td>Enable this option if you want each selected item to keep the same relative position to each other while snapping. This is the default. Disable this option to snap each selected item to the grid independently.</td>
-</tr>
-<tr>
-<td colspan="2"><strong>Snap Axis</strong></td>
-<td>Choose how vertices snap to the grid while moving.</td>
-</tr>
-<tr>
-<td></td>
-<td><strong>Active Axis</strong></td>
-<td>Vertices snap only along the currently active axis. This is the default.</td>
-</tr>
-<tr>
-<td></td>
-<td><strong>All Axes</strong></td>
-<td>Vertices snap to all axes simultaneously.</td>
-</tr>
-</tbody>
-</table>
-
-
-
-<a name="toolbar"></a>
-
-## Toolbar
-
-Use these properties to set default behavior for the [ProBuilder toolbar](toolbar.md).
-
-![Toolbar section](images/prefs_toolbar.png)
-
-| **Property:** | **Description:** |
+| **Property** | **Function** |
 | :--- | :--- |
-| __Shift Key Tooltips__ | Enable this option to only show tooltips when the mouse cursor is hovering over a button and you are holding down **Shift**.<br/>By default, tooltips appear when the mouse cursor hovers over a button for more than a second. |
-| <a name="icongui"></a>__Icon GUI__ | Enable this option to use toolbar buttons that [display icons only](toolbar.md#buttonmode). <br />Disable this option to use toolbar buttons that [display text only](toolbar.md#buttonmode).<br />**Note:** You can also [use the context menu](customizing.md#buttons) to switch between icons and text. |
-| <a name="toolbarloc"></a>__Toolbar Location__ | Choose the location where you want the [Edit Mode toolbar](edit-mode-toolbar.md) to appear in the Scene view. Possible locations are:<br /><br />- **Upper Center**<br />- **Upper Left**<br />- **Upper Right**<br />- **Bottom Center**<br />- **Bottom Left**<br />- **Bottom Right** |
-
-
-
-
-
-
-<a name="uvs"></a>
+| **Snap As Group** | All selected items keep the same relative position to each other while snapping. This is the default. Disable this option to snap each selected item to the grid independently. |
+| **Snap Axis** | Choose how vertices snap to the grid when you move them. **Active Axis**: Vertices snap only along the currently active axis [[Jon: what does "currently active" mean?]]. This is the default. **All Axes**: Vertices snap to all axes simultaneously. |
```

Comment:
The active axis is the one you're selecting right now. When you activate the Move tool, you get the 3 little arrows: a green one for the Y-axis, a red one for the X-axis, and a blue one for the Z-axis. Whichever one you select is the active axis.

Reply:
This is mostly used for vertex snapping (which you activate by holding **V**).

Reply:
But you can also select the "middle" and move freely. What happens then?

Reply:
That just moves it freely, as you say. @modrimkus, any insight into this one?

Reply:
While this involves element (vert/edge/face) snapping, it's not related to the vertex-vertex snapping (i.e. holding V). This pref is strictly for how elements snap to the grid (when the editor's built-in absolute grid snapping is enabled). 

- Active axis. If element is moved along an axis (constrained to currently active axis), then snapping will only take place on that axis (places where that axis hits a grid line or point).
- All axes. Even if element is moved along an axis, snapping will ignore that and pick the nearest point on grid and snap to it. 
- If in Free move (i.e. user holds shift and uses the square manipulator), snapping falls back to "All axes" no matter which preference.

Video illustrating the Active Axis vs All Axes setting.
https://github.com/user-attachments/assets/3fcbd0bd-3577-4f3b-a9b0-cf269d990107

Reply:
```suggestion
| **Snap Axis** | Choose how mesh elements snap to the grid when you move them (if Snap to Grid is active). **Active Axis**: When you move an element along an axis, the element snaps to the next grid point on that axis. This is the default. **All Axes**: An element snaps to the nearest grid point, even if it's on a different axis to the one you're moving on. |
```